### ISSUE TITLE
Easier to understand, info icon for resources and fields

### ISSUE THREAD

**Issue by illume** [illume/headlamp#154]
## Current situation

Kubernetes jargon is confusing and extensive. Making it hard to understand for users new and long term.

Like many usability issues, it's also an accessibility issue. See WCAG 2.1, which has requirements for plain language and has guidelines for Jargon.

For languages other than English we are currently using the English Kubernetes terms. This is for two reasons. Firstly there aren't always appropriate words, so the English word can be better. Secondly, even when there are words available they can be confusing when much of the tooling using the English terms only.

## Impact

Headlamp (and Kubernetes) will become more usable and accessible.

## Ideal future situation

There will be a way for K8s Jargon words to be understood by people without extensive Kubernetes jargon knowledge.

## Implementation options

WCAG 2.1 and has some techniques about what to do when things like jargon are used. https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=313#unusual-words

Kubernetes maintains a glossary here: https://kubernetes.io/docs/reference/glossary/?all=true
Some parts are translated, but not all that much. The French translation is https://kubernetes.io/fr/docs/reference/glossary/?all=true is one of the more complete glossary translations, but still misses much.

The translations inside Headlamp also have a list of non-translated k8s English terms. The existing Documentation tab should be considered to see if it can provide other language information.

—-
I want there to be an info icon for each resource, next to the title and where the add button is on each page.

Also, for each of the fields I want there to be a way to hover over the field in order to find more information about the field. This should be unobtrusive, and no visual stuff there by default.

Maybe info icon is a button that can turn on this hover mode. Or it can be on all the time.

Also it should be a11y friendly.

The texts should come from the kubernetes docs where possible, and there should be some scripts to keep the text in sync. That is run, and download new texts from the kubernetes website.
